README.md

```diff
@@ -20,6 +20,53 @@ Modern vision of the mobile application for the Open edX platform from Raccoon G
 
 6. Click the **Run** button.
 
+## Translations
+
+### Getting Translations for the App
+Translations aren't included in the source code of this repository as of [OEP-58](https://docs.openedx.org/en/latest/developers/concepts/oep58.html). Therefore, they need to be pulled before testing or publishing to App Store.
+
+Before getting the translations for the app we need to install the requirements put inside the `requirements.txt` in the `i18n_scripts`. This can be done easily by running the following make command:
```

Comment:
```suggestion
Before retrieving the translations for the app, we need to install the requirements listed in the requirements.txt file located in the i18n_scripts directory. This can be done easily by running the following make command:
```